guardrails/src/adapters/runtime/nemo_runtime.py

```
@@ -0,0 +1,147 @@
+
+                new_rails = LLMRails(rails_config)
+
+                # Atomic swap
+                self._rails = new_rails
+                self._revision = materialized.revision
+
+                logger.info(
+                    f"Successfully loaded guardrail configuration revision {materialized.revision}"
+                )
+
+            except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)